test/unittests/mongoBackend/mongoUpdateContextSubscription_test.cpp

```diff
@@ -272,7 +272,7 @@ static void prepareDatabase(void) {
                         "lastNotification" << 55000000 <<
                         "throttling" << 10 <<
                         "reference" << "http://notify5.me" <<
-                        "format" << "JSON" <<
+                        "format" << "legacy" <<
```

Comment:
This line here is the reason everything works.
I guess my grep was in combination with something else, like a function call or something ...